.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
We already have an issue template https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/.github/issue_template.md.

Reply:
I wanted to split this into bug and feature request. I hope it makes sense.